pkg/apis/kops/componentconfig.go

```diff
@@ -553,6 +553,10 @@ type KubeSchedulerConfig struct {
 	UsePolicyConfigMap *bool `json:"usePolicyConfigMap,omitempty"`
 	// FeatureGates is set of key=value pairs that describe feature gates for alpha/experimental features.
 	FeatureGates map[string]string `json:"featureGates,omitempty" flag:"feature-gates"`
+	// The maximum qps to send to the api server
```

Comment:
Maybe we should create a different struct for the config flags? This would draw a clear line of where the functionality is used.